File: ReactNativeLearn/ios/RNBridge.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(RNBridge, NSObject)

RCT_EXTERN_METHOD(addEvent:(NSString *)name location:(NSString *)location date:(nonnull NSNumber *)date)

RCT_EXPORT_METHOD(constantsToExport:(RCTResponseSenderBlock)callback)
{
  NSArray *events = @[@21,@2,@3];
  callback(events);
}

@end

File: ReactNativeLearn/ios/RNBridge.swift
//
//  RNBridge.swift
//  ReactNativeLearn
//
//  Created by runlong on 2019/8/28.
//  Copyright © 2019 Facebook. All rights reserved.
//

import UIKit

@objc(RNBridge)
class RNBridge: NSObject {
  @objc(addEvent:location:date:)
  func addEvent(name: String, location: String, date: NSNumber) -> Void {
    // Date is ready to use!
    print("name:\(name),location:\(location),date:\(date)")
  }
  
  @objc
  func constantsToExport() -> [String: Any]! {
    return ["someKey": "someValue"]
  }
}

